cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -765,7 +765,7 @@ func (c *client) waitForKubectlApply(manifest string) error {
 		klog.V(2).Infof("Waiting for kubectl apply...")
 		err := c.kubectlApply(manifest)
 		if err != nil {
-			if err.Error() == io.EOF.Error() || strings.Contains(err.Error(), "refused") {
+			if strings.Contains(err.Error(), io.EOF.Error()) || strings.Contains(err.Error(), "refused") {
```

Comment:
Is this because the error is wrapped? Can you compare against `errors.Cause(err)` instead?

Reply:
Unfortunately no :/ the error is a string that comes from kubectl, which this client calls using cmd.Run